includes/Admin.php

```diff
@@ -258,6 +258,15 @@ public function add_tpc_editor_data( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Use the Neve builtin compatibility to check for specific support.
+	 *
+	 * @return bool
+	 */
+	private function neve_theme_has_support( $feature ) {
+		return defined( 'NEVE_COMPATIBILITY_FEATURES' ) && isset( NEVE_COMPATIBILITY_FEATURES[ $feature ] );
```

Comment:
One curiosity, in case in future we decide to remove some stuff and mark those flags as false, shouldn't we also check if they're true? Or in those case, we would just remove the items instead of marking them false?

Reply:
All checks at the moment assume that the features are not temporary. In the future if required I think the key will just be removed to default to a different behaviour.

Reply:
Makes sense, thank you.